schemas/telemetry/configuration-schema.json

```diff
@@ -0,0 +1,186 @@
+                  "minimum": 0,
+                  "maximum": 100
+                },
+                "trace_sample_rate": {
+                  "type": "integer",
+                  "description": "The percentage of requests traced",
+                  "minimum": 0,
+                  "maximum": 100
+                },
+                "premium_sample_rate": {
+                  "type": "integer",
+                  "description": "The percentage of sessions with Browser RUM & Session Replay pricing tracked (deprecated in favor of session_replay_sample_rate)",
+                  "minimum": 0,
+                  "maximum": 100
+                },
+                "replay_sample_rate": {
+                  "type": "integer",
+                  "description": "The percentage of sessions with Browser RUM & Session Replay pricing tracked (deprecated in favor of session_replay_sample_rate)",
+                  "minimum": 0,
+                  "maximum": 100
+                },
+                "session_replay_sample_rate": {
+                  "type": "integer",
+                  "description": "The percentage of sessions with Browser RUM & Session Replay pricing tracked",
+                  "minimum": 0,
+                  "maximum": 100
+                },
+                "use_proxy": {
+                  "type": "boolean",
+                  "description": "Whether a proxy configured is used"
+                },
+                "use_before_send": {
+                  "type": "boolean",
+                  "description": "Whether beforeSend callback function is used"
+                },
+                "silent_multiple_init": {
+                  "type": "boolean",
+                  "description": "Whether initialization fails silently if the SDK is already initialized"
+                },
+                "track_session_across_subdomains": {
+                  "type": "boolean",
+                  "description": "Whether sessions across subdomains for the same site are tracked"
+                },
+                "use_cross_site_session_cookie": {
+                  "type": "boolean",
+                  "description": "Whether a secure cross-site session cookie is used"
+                },
+                "use_secure_session_cookie": {
+                  "type": "boolean",
+                  "description": "Whether a secure session cookie is used"
+                },
+                "action_name_attribute": {
+                  "type": "string",
+                  "description": "Attribute to be used to name actions"
+                },
+                "use_allowed_tracing_origins": {
+                  "type": "boolean",
+                  "description": "Whether the allowed tracing origins list is used"
+                },
+                "default_privacy_level": {
+                  "type": "string",
+                  "description": "Session replay default privacy level",
+                  "enum": ["mask-user-input", "mask", "allow"]
```

Comment:
<!-- Thoughts represent an idea that popped up from reviewing. These comments are non-blocking by nature, but they are extremely valuable and can lead to more focused initiatives and mentoring opportunities.  -->
**💭 thought:** I am wondering if listing the values here may not add more disturbance than value.
When adding new values, it would require us to also updates things here. 
Do you see a benefit for listing the allowed values here?
(same for other enum)
‏

Reply:
Hum, it's true that maintaining these values can be painful. I'll remove them